OneBusAway/otpkit
# Add tripplanner and map overlay

## Description

This PR includes adding selectable direction in trip planner 

## Related Issues
#34

## Media
https://github.com/user-attachments/assets/db2ce2d7-f729-4a69-86e4-5bb0468c32b3

## Test Instructions

1. Select Origin and destination. It can be from search location, current location, and direct map selection
2. Select itinerary
3. See the map overlay. Check if it's visible
